recipes/ectyper/meta.yaml

```diff
@@ -0,0 +1,30 @@
+about:
+    license: MIT
+    summary: Ectyper is a python script for serotyping e.coli genomes
+    author: Camille La Rose, Chad Laing, Sam Sung
+    home: https://github.com/phac-nml/ecoli_serotyping
+
+package:
+    name: ectyper
+    version: 0.0.8
+
+source:
+    url: https://github.com/phac-nml/ecoli_serotyping/archive/v0.1.tar.gz
+    md5: 06b5793ed0f7000a05e20600b1ac5d0c
+
+requirements:
+    build:
+        - python 3.6.3.*
+    run:
+        - python 3.6.3.*
+        - pandas 0.21.0.*
```

Comment:
All other pins are here very strict, is this really needed? Biopython for example is stable afaik and does not need to be pinned to 1.69.